### ISSUE TITLE
Why do you trained a model build from scratch instead of fine-tuning a pretrained model？

### ISSUE THREAD

**Issue by Sumching** [ashafahi/free_adv_train#3]
Hi，
I notice that you did not use a pretrained model. Is it too hard to train? Or is its performation not good?
Thanks.

**Comment by ashafahi** [comment]
Hi,
Correct. We have started with random initialization. However, starting from a pre-trained net also works. It can even increase clean validation accuracy a bit.
